packages/suite/src/hooks/guide/useGuideLoadArticle.ts

```diff
@@ -5,7 +5,7 @@ import type { GuideNode } from '@suite-common/suite-types';
 import type { Locale } from 'src/config/suite/languages';
 
 export const loadPageMarkdownFile = async (id: string, language = 'en'): Promise<string> => {
-    const file = await import(`@trezor/suite-data/files/guide/${language}${id}`);
+    const file = await import(/* @vite-ignore */ `@trezor/suite-data/files/guide/${language}${id}`);
```

Comment:
This import was absolutely okay, but that dynamic import with relative paths was broken. Weird.

Reply:
resolved, the file should be imported with full path along with extension.